DSE_6.9_Release_Notes.md

```diff
@@ -2,6 +2,35 @@
 DSE 6.9.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any.
 Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# Release notes for 6.9.10
+12 June 2025
+
+## Components versions for DSE 6.9.10
+ * Apache Solr™ 6.0.1.5.2977
+ * Apache Spark™ 2.5.0.6
+ * Apache TinkerPop™ 3.4.15-20250603-0956e859&ast;
+ * Apache Tomcat® 8.5.100
+ * DSE Java Driver 1.10.0-dse-20241015 (DSE *internal-only* version)
+ * Netty 4.1.119.1.dse&ast;
+ * Spark JobServer 0.8.0.57&ast;
+
+**NOTE**: above-listed DSE Java Driver is an _internal-version_ only.
+If you're developing applications, please refer to the [Java Driver documentation](https://docs.datastax.com/en/driver-matrix/doc/java-drivers.html) to choose an appropriate version.
+
+## 6.9.10 DSE Core
+* Updated the Java Development Kit (JDK) versions to `8u452` and `11.0.27`. These JDKs help build and test DSE, and are available in DSE Docker images. The following exceptions for DSE UBI images apply:. (DSP-24710)
+
+## 6.9.10 DSE Cassandra
+* Removed non-supported Java 8 configuration files. (DSP-24845)
+
+## 6.9.10 DSE CVE
+* Upgraded the `net.minidev:json-smart` Java JSON parser package to version `2.5.2` to resolve a Denial of Service (DoS) vulnerability. (DSP-24851, [CVE-2024-57699](https://nvd.nist.gov/vuln/detail/CVE-2024-57699))
```

Comment:
@rh-datastax I'm wondering if we need to add "resolve a DoS vulnerability" here. This is already categorized as a CVE and we should provide as little information as possible on these to prevent malicious actors to attempt exploiting previous versions. Any thoughts?